components/button/demo/floating-buttons-page.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
```

Comment:
This file is just used within the iframes floating-buttons demo and is not linked to from index.html. I grouped the always float demo and the append content etc demo together in demo snippets in `floating-demos.html` to try to simplify a bit.